css/main.css

```
@@ -335,13 +329,61 @@ a {
 }
 
 table {
-  display: inline-block;
-  border-spacing: 0;
-  border-collapse: collapse;
-  overflow-x: auto;
-  max-width: 100%;
+  display: grid;
+  width: 100%;
   text-align: left;
-  vertical-align: top;
+  vertical-align: middle;
+  background: #001634;
+  color: #eee;
+  border-collapse: collapse;
+}
+
+thead {
+  overflow: hidden;
+  width: 100%;
+  min-width: 500px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+}
```

I can only see horizontal scroll being supported on mobile view, not sure if thats the intention but we should also support vertical scrolling. This will require a height/max-height and overflow property on the `table` element and `overflow: hidden;` needs to be removed from `thead` element to avoid clipping of the table head.
```suggestion
table {
  display: grid;
  width: 100%;
  height: 300px;
  text-align: left;
  vertical-align: middle;
  background: #001634;
  color: #eee;
  border-collapse: collapse;
  overflow: scroll;
}

thead {
  width: 100%;
  min-width: 500px;
  border-top-left-radius: 5px;
  border-top-right-radius: 5px;
}
```
We can modify the scroll bar later if required.

---

Will implement the vertical scroll.
The `overflow: hidden` on thead won't affect the content of thead but only its border-radius, so that should be fine I believe

---

A vertical scroll implementation will also require a sticky `<th>` implementation to ensure people don't lose track of the column they're on.
I don't yet see a compelling use-case for vertical scroll - We'll obviously use `@media` queries to ensure it works only on mobile, but a sticky `<th>` seems like a good idea regardless of whether or not vertical scroll is implemented. Thoughts?

---

Sounds good!.. I'll try it out and commit.